traffic_ops/app/db/migrations/20180312000000_create_config_diffs.sql

```diff
@@ -0,0 +1,40 @@
+/*
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+--
+-- Name: config_diffs; Type: TABLE; Schema: public; Owner: traffic_ops
+--
+
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE config_diffs (
+    config_id bigserial NOT NULL PRIMARY KEY,
```

Comment:
any reason this is not just `id`?

Reply:
no functional reason

Reply:
every other table uses `id` -- should stick with that...

Reply:
ok, can do